jib-plugins-extension-common/src/main/java/com/google/cloud/tools/jib/plugins/extension/JibPluginExtension.java

```diff
@@ -0,0 +1,20 @@
+/*
+ * Copyright 2020 Google LLC.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package com.google.cloud.tools.jib.plugins.extension;
+
+/** Base type for Jib plugin extensions. */
+public interface JibPluginExtension {}
```

Comment:
this is used just to indentify the services to load?

Reply:
Not really, but at least this type is required in the shared `JibPluginExtensionException` whose constructors force to pass an extension class.